src/init.ts

```diff
@@ -84,7 +84,9 @@ export const init = async (): Promise<void> => {
     );
 
     // Inject CSS into the application
-    await injectCss(settings.css || 'cm.css');
+    // TODO: This should probably be awaited but it breaks non-stubbed airgap inits
+    // https://transcend.height.app/T-35913
+    injectCss(settings.css || 'cm.css');
```

Comment:
when can then() this, but if we weren't awaiting before it's likely fine. the side effect could be a FOUC though

Reply:
we never actually deployed the cm ui version change that had this, and we havent gotten any reports of obvious stylesheet loads so im pretty sure its fine

Reply:
ok sgtm